### PR TITLE
SH - Updated CONTRIBUTE guidance

### PR DESCRIPTION
**Pull request by SamHollings**
# Description

❓**What**: Updated the CONTRIBUTE guidance

🧠**Why?**: It was a bit old, and not entirely correct

👨‍💻**How?**: Made it reference codespaces, and venvs rather than conda.

# Checklist:
Have checked for the following:
- [ ] The website still builds correctly, and you can view it using `mkdocs serve`.
- [ ] There are no new "warnings" from mkdocs
- [ ] Does your page follow the [page template](https://nhsdigital.github.io/rap-community-of-practice/example_RAP_CoP_page/) (or [here in Markdown](https://github.com/NHSDigital/rap-community-of-practice/blob/main/docs/example_RAP_CoP_page.md))?
- [ ] Spelling errors
- [ ] Consistent capitalization
- [ ] Consistent numbers
- [ ] Material features incorrectly implemented: search for code blocks and markers (e.g. !!!).
- [ ] Code snippets don't work
- [ ] Images not working
- [ ] Links not working

## Where it was tested

- Github Codespaces - 2-core, 4GB RAM, 32GB hard drive
- devcontainer.json describes further settings

### CODE REVIEW

**Comment by SamHollings** [comment]
I didn't do any of the usual checks, but then I haven't edited any of the main files of the website.

**Review by warren-davies4 — APPROVED** [review]
great updates but we should also probably remove this page:

https://nhsdigital.github.io/rap-community-of-practice/site_info/rap-release-workflow/

File: docs/site_info/rap-release-workflow.md [hunk 1]
```diff
@@ -7,6 +7,32 @@ tags:
...
+    B-->C[pull request checked];
+    C-->F[approved and changes merged into the main branch];
+    C--Feed back given-->E[Make required changes];
+    E-->C;
+    F--At a significant milestone-->D[a release is made, bumping up the version of the repo and updating the visible website];
+```
+
+It's important to note that the website is not built off the "main" branch - but instead it builds "on release".
+
+This is described fully in the github action that builds the website: [pages-build-deployment](.github/workflows/pages-build-deployment.yml)
```
Comment (warren-davies4) [inline docs/site_info/rap-release-workflow.md:32]:
```suggestion
This is described fully in the github action that builds the website: [pages-build-deployment](https://github.com/NHSDigital/rap-community-of-practice/blob/main/.github/workflows/pages-build-deployment.yml)
```
```suggestion
This is described fully in the github action that builds the website: [pages-build-deployment](https://github.com/NHSDigital/rap-community-of-practice/blob/main/.github/workflows/pages-build-deployment.yml)
```

File: docs/site_info/rap-release-workflow.md [hunk 2]
```diff
@@ -7,6 +7,32 @@ tags:
 
 #
 
+## Current Release Process (coding in the open)
+
+!!! warning "The release process below is no longer in use!"
```
Comment (warren-davies4) [inline docs/site_info/rap-release-workflow.md:12]: Your call but consider moving this warning below the "Old release process" header, when I first looked at this, I thought it referred to the new content

**Review by warren-davies4 — APPROVED** [review]

**Pull request merged**